.travis.yml

```diff
@@ -15,6 +15,7 @@ cache:
     - dev-packages/cli/node_modules
     - examples/api-samples/node_modules
     - examples/browser/node_modules
+    - examples/electron-api-samples/node_modules
```

Comment:
do we really need a new package? Could we add the second extension entry point in `api-samples` instead?

Reply:
Sure, why is it a problem having a dedicated sample for electron?

Reply:
@marechal-p, please ping me when you're done, and I move the sample.

Reply:
@kittaakos it is all yours.

Reply:
I am not sure if we want to move it to the existing sample project because if we move it to there, the browser example will depend on it, which pulls the electron dependency in. Let me check, if it works, I will move it. Otherwise, I'll leave here as originally wanted.